web/src/cypress/support/graphql.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint no-unused-expressions: 0 */
```

Comment:
Let's rewrite those two assertions so we don't need to disable the rule.

The first is just to display the message as a failure. A better way may be to do `expect.fail(data.errors[0].message)`

Second can be `expect(data).to.not.have.property('errors')` I think.

Do a negative test locally to make sure the test/failure output is useful